packages/flutter/lib/src/widgets/app.dart

```diff
@@ -820,7 +820,7 @@ class WidgetsApp extends StatefulWidget {
   ///
   /// To provide a localized title instead, use [onGenerateTitle].
   /// {@endtemplate}
-  final String title;
+  final String? title;
```

Comment:
It's odd that WidgetsApp.title and MaterialApp.title / CupertinoApp.title have different defaults (empty string vs null). Those should probably stay in sync.

Reply:
Oops, seems like I missed the `WidgetsApp.title` ones? Done.

Great catch!